### PR TITLE
Js implemented in mobile menu

### PR DESCRIPTION
**Pull request by HossainAraf**
We have implemented the following features  in mobile menu:

- When the user clicks (or taps) the hamburger button, the mobile menu appears. [Code](https://github.com/HossainAraf/Portfolio-Mobile-First/blob/c75452596bd3ab6896d67158917ab12d0add2df9/app.js#L8C1-L25C2)
- When the user clicks (or taps) the close (X) button, the mobile menu disappears. [Code-1](https://github.com/HossainAraf/Portfolio-Mobile-First/blob/c75452596bd3ab6896d67158917ab12d0add2df9/app.js#L10C1-L10C1) [Code-2](https://github.com/HossainAraf/Portfolio-Mobile-First/blob/c75452596bd3ab6896d67158917ab12d0add2df9/app.js#L27)
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears. [Code](https://github.com/HossainAraf/Portfolio-Mobile-First/blob/c75452596bd3ab6896d67158917ab12d0add2df9/app.js#L36C1-L41C1)
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. [Code](https://github.com/HossainAraf/Portfolio-Mobile-First/blob/c75452596bd3ab6896d67158917ab12d0add2df9/index.html#L21-L22)

### CODE REVIEW

**Review by Strangeal — COMMENTED** [review]
Hi @HossainAraf, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] It appears that the project was intended to be completed using pair programming. However, based on the commit history, it seems that you completed the project alone instead of collaborating with your coding partner. For the purpose of maintaining the integrity of the pair programming process, I kindly request you to reach out to your coding partner and work together on the project before resubmitting it for review.  

  - [ ] In case you encountered any difficulties or issues with your coding partner that prevented you from collaborating, I encourage you to contact Student Success ([Success@microverse.org](mailto:Success@microverse.org)) to address the situation and seek assistance. It's important to adhere to the principles of pair programming and communicate any challenges you may face in order to ensure a fair and effective learning environment.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by HossainAraf** [comment]
Dear @Strangeal, 
Perhaps you missed that one of my coding partners participated. Please, find the commit [link](https://github.com/HossainAraf/Portfolio-Mobile-First/pull/10/commits/615d51635103457d9fea467255d92e9149c87365) 
So, Please proceed with reviewing the file. Thanks.

**Comment by nahnahdev** [comment]
## Project Status: INVALIDATED :red_circle: 
Hi @HossainAraf , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

This is because you have submitted a project, that is missing the `.eslintrc.json file` which is in charge of the `Eslint` checks or JAVASCRIPT LINTER CHECKS so kindly ensure that you have properly installed the [HTML, CSS, AND JAVASCRIPT LINTERS](https://github.com/microverseinc/linters-config/tree/master/html-css-js) in your project :100: 
- Your Current checks 
image

- What you should have
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
       <div class="toggle-button" type="toogle">
-        <span class="menu-icon"><img src="img/Menu.png" alt="menu"></span>
+        <i class="fa fa-bars menu-icon"></i>
       </div>
       <ul class="menu">
-        <li><a href="#" title="Go to title">About</a></li>
-        <li><a href="#" title="Go to services">Services</a></li>
-        <li><a href="#" title="Go to contact">Contact</a></li>
+        <li class="menu-item"><a href="#about-myself" title="Go to title">About</a></li>
+        <li class="menu-item"><a href="#work" title="Go to services">Services</a></li>
```
Comment (divinecharlotte) [inline index.html]: - [ ] kindly consider sticking to the design as much as possible by adding the right `text` as the one given in the design to stay consistence with the requirement below 👇 
> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Screen Shot 2023-06-28 at 06 16 58

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Descriptive pr 👍 
- Descriptive readme 👍 

 




 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -14,13 +14,14 @@
...
-        <span class="menu-icon"><img src="img/Menu.png" alt="menu"></span>
+        <i class="fa fa-bars menu-icon"></i>
       </div>
       <ul class="menu">
-        <li><a href="#" title="Go to title">About</a></li>
-        <li><a href="#" title="Go to services">Services</a></li>
-        <li><a href="#" title="Go to contact">Contact</a></li>
+        <li class="menu-item"><a href="#about-myself" title="Go to title">Portfolio</a></li>
+        <li class="menu-item"><a href="#work" title="Go to services">About</a></li>
+        <li class="menu-item"><a href="#form" title="Go to contact">Contact</a></li>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure the `About` link directs the user to `about-myself` and `Portfolio` directs to the `work` part of the page.

  ```suggestion
        <li class="menu-item"><a href="#work" title="Go to title">Portfolio</a></li>
        <li class="menu-item"><a href="#about-myself" title="Go to services">About</a></li>
        <li class="menu-item"><a href="#form" title="Go to contact">Contact</a></li>
```

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @HossainAraf   , :wave:
### I'm @levy002, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!




## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @HossainAraf 

Your project is complete! Great job🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Figma design well followed
- works as expected

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**